man/new_city.Rd

```diff
@@ -14,6 +14,10 @@ new_city(name = NULL, country = NULL, lat = NULL, long = NULL)
 \item{lat}{A single numeric value to be used as the latitude.}
 
 \item{long}{A single numeric value to be used as the longitude.}
+
+\item{method}{A character string specifying the geocoding method to use. Defaults to "osm" (OpenStreetMap). Supported methods include:
```

Comment:
`...to use when the user does not specify \code{lat} or \code{long}.`

Reply:
Yes better!